README.md

```diff
@@ -60,3 +60,21 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing this app
+
+Before you test this app you need to get your environment ready. To do that install pytest:
+```bash
+poetry add pytest
```

Comment:
Note that you've run this command which stores pytest as a dependency (see your pyproject.toml file) and therefore another developer will automatically have it in their virtual environment once they run `poetry install` - so this step isn't needed

Reply:
But it would be nice to also mention dependencies, like the user having geckodriver downloaded and available on their path, as well as the requirement to have Firefox

Reply:
Done, thanks.